process/block/shardblock.go

```diff
@@ -1726,9 +1726,13 @@ func (sp *shardProcessor) createMiniBlocks(
 
 // ApplyBodyToHeader creates a miniblock header list given a block body
 func (sp *shardProcessor) ApplyBodyToHeader(hdr data.HeaderHandler, bodyHandler data.BodyHandler) (data.BodyHandler, error) {
-	log.Trace("started creating block header",
-		"round", hdr.GetRound(),
-	)
+	tm := core.NewTimeMeasure()
+	tm.Start("ApplyBodyToHeader")
+	defer func() {
+		tm.Finish("ApplyBodyToHeader")
+
+		log.Debug("time measurements", tm.GetMeasurements()...)
```

Comment:
in peer processor you put `"measurements checkForMissedBlocks"` for log title. Why don't use use the same pattern here ? It would be easier when looking over the logs

Reply:
done